app/controllers/station_images_controller.rb

```diff
@@ -0,0 +1,30 @@
+class StationImagesController < ApplicationController
+  def create
+    @image = StationImage.new(station_image_params)
+
+    if @image.save
+      flash[:notice] = "Image posted"
+      redirect_to station_path(@image.station_id)
+
+    else
+      flash[:notice] = "Upload failed"
+
+      @station = Station.find(params[:station_id])
+      @user = current_user
+
+      @reviews = @station.reviews
+      @review = Review.new
+
+      @votes = @review.votes
+
+      @errors = @image.errors.full_messages
+      render "stations/show"
```

Comment:
You never use @errors in your show:

<% if @review.errors.any? %>
  <% @review.errors.full_messages.each do |error| %>
    <%= error %>
  <% end %>
<% end %>

I would either delete the @errors or use it in view